Remotion/Core/Core/Utilities/StringUtility.cs

```diff
@@ -535,4 +540,4 @@ public ParseException (SerializationInfo info, StreamingContext context)
     {
     }
   }
-}
\ No newline at end of file
+}

```

Comment:
This happens when doing a suggestion commit via Github, dunno if we keep this.

Reply:
don't care